src/status_im/ui/screens/contacts_list/views.cljs

```diff
@@ -21,41 +21,44 @@
-    :accessibility-label :add-new-contact-button
-    :on-press            #(re-frame/dispatch [:navigate-to :new-contact])}])
-
 (defview contacts-list []
   (letsubs [blocked-contacts-count [:contacts/blocked-count]
-            contacts      [:contacts/active]]
-    [react/view {:flex 1}
-     [topbar/topbar {:title (i18n/label :t/contacts)}]
-     [react/scroll-view {:flex 1}
-      [add-new-contact]
-      (when (pos? blocked-contacts-count)
-        [react/view {:margin-vertical 16}
-         [quo/list-item
-          {:title               (i18n/label :t/blocked-users)
-           :icon                :main-icons/cancel
-           :theme               :negative
-           :accessibility-label :blocked-users-list-button
-           :chevron             true
-           :accessory           :text
-           :accessory-text      blocked-contacts-count
-           :on-press            #(re-frame/dispatch [:navigate-to :blocked-users-list])}]])
-      (if (seq contacts)
-        [list.views/flat-list
-         {:data                      contacts
-          :key-fn                    :address
-          :render-fn                 contacts-list-item}]
-        [react/view {:align-items :center :flex 1 :justify-content :center}
-         [react/text {:style {:color colors/gray :margin-vertical 24}}
-          (i18n/label :t/you-dont-have-contacts)]
-         [invite/button]])]]))
+            contacts [:contacts/active]]
+    [:<>
+     (when (pos? blocked-contacts-count)
+       [react/view {:margin-vertical 16}
+        [quo/list-item
+         {:title               (i18n/label :t/blocked-users)
+          :icon                :main-icons/cancel
+          :theme               :negative
+          :accessibility-label :blocked-users-list-button
+          :chevron             true
+          :accessory           :text
+          :accessory-text      blocked-contacts-count
+          :on-press            #(re-frame/dispatch [:navigate-to :blocked-users-list])}]])
+     (if (seq contacts)
+       [list.views/flat-list
+        {:data      contacts
+         :key-fn    :address
+         :render-fn contacts-list-item
+         :footer    [react/view {:height 68}]}]
+       [react/view {:padding-horizontal 32 :margin-top 32}
+        [react/view {:border-width               1
+                     :border-color               colors/gray-lighter
+                     :border-top-right-radius    16
+                     :border-bottom-left-radius  16
+                     :border-top-left-radius     16
+                     :border-bottom-right-radius 4
+                     :padding-horizontal         12
+                     :padding-vertical           6}
+         [react/text {:style {:color colors/gray :line-height 22}}
+          (i18n/label :t/contacts-descr)]]
+        [react/touchable-highlight {:on-press #(re-frame/dispatch [:navigate-to :new-contact])}
+         [react/view {:flex-direction :row :align-items :center :align-self :center
```

Comment:
That's a comment for all such places in PR: imo if you want to write multiple key pairs in a single line, let's divide them with commas. Otherwise let's put a single pair per line. The reason is simple, it is hard for reading (especially when both key and value are of the same type).

Reply:
ok

Reply:
@rasom should be fixed